### PR TITLE
Remove advertising sidebar

### PR DESCRIPTION
**Pull request by tbepdb**
Advertising on canvas consumes a high amount of CPU time.

### CODE REVIEW

File: integrate.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ WebApp._onPageReady = function()
 {
     // Connect handler for signal ActionActivated
     Nuvola.actions.connect("ActionActivated", this);
+    document.querySelector(".centerblock-wrapper").setAttribute('style', 'width: 100%');
```
Comment (jiri-janousek) [inline integrate.js]: `document.querySelector()` may return `null` and `.setAttribute` would throw an error in that case. Either check the return value of  `document.querySelector()` not to be `null` or add `try { ...} catch (e){ /* ignored */ }`.

File: integrate.js [hunk 2]
```diff
@@ -65,6 +65,9 @@ WebApp._onPageReady = function()
...
+    var sidebar = document.querySelector(".sidebar__placeholder .sidebar");
+    sidebar.parentNode.removeChild(sidebar);
```
Comment (jiri-janousek) [inline integrate.js]: The same here. The `sidebar` variable may be `null` and throw an exception. You can use a single try-catch block.

File: integrate.js [hunk 3]
```diff
@@ -2,14 +2,14 @@
  * Copyright 2015 Jiří Janoušek <janousek.jiri@gmail.com>
  *
  * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are met: 
- * 
+ * modification, are permitted provided that the following conditions are met:
+ *
```
Comment (jiri-janousek) [inline integrate.js:6]: Please do not alter white space.

**Review by jiri-janousek — CHANGES_REQUESTED** [review]
I'm happy to merge you code as soon as you address my comments from code review.

File: integrate.js (hunk 3 above)
Comment (tbepdb) [reply integrate.js:6]: Sorry, its my editor settings.

**Comment by jiri-janousek** [comment]
@tbepdb,you didn't have to close the PR. A cleaner way is to force-overwrite he current one with `git push -f` and leave a comment about it..

**Pull request closed without merging**